core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -63,6 +63,7 @@
 // The test suite itself is Serializable so that anonymous Function implementations can be
 // serialized, as an alternative to converting these anonymous classes to static inner classes;
 // see http://stackoverflow.com/questions/758570/.
+@SuppressWarnings("serial")
```

Comment:
The warning is bogus though, about serialVersionUID? it should actually not be specified by default. Does the Spark build actually enable this warning though, or is it just Eclipse? I didn't think -Xlint was on.

This can't be the only warning :) I see a load still, that I've been meaning to fix again again.